### PR TITLE
Vitamin Cheat Sheet: Improve Vitamin Sources and Benefits

### PR DESCRIPTION
**Pull request by MrChrisW**
Updated the vitamin cheat sheet to include additional sources (vegan options) and changed benefit descriptions. 

____
**Sources:**

- Descriptions: http://visual.ly/vitamins-cheat-sheet
- Vitamin A: https://ods.od.nih.gov/factsheets/VitaminA-HealthProfessional/
- Vitamin B1: https://www.nlm.nih.gov/medlineplus/bvitamins.html
- Vitamin B2: http://umm.edu/health/medical/altmed/supplement/vitamin-b2-riboflavin
- Vitamin B3: http://umm.edu/health/medical/altmed/supplement/vitamin-b3-niacin
- Vitamin B6: https://ods.od.nih.gov/factsheets/VitaminB6-HealthProfessional/
- Vitamin B12: https://ods.od.nih.gov/factsheets/VitaminB12-HealthProfessional/
- Vitamin C: https://ods.od.nih.gov/factsheets/VitaminC-HealthProfessional/
- Vitamin D: https://ods.od.nih.gov/factsheets/VitaminD-HealthProfessional/
- Vitamin E: https://ods.od.nih.gov/factsheets/VitaminE-HealthProfessional/
- Vitamin K: https://ods.od.nih.gov/factsheets/VitaminK-HealthProfessional/

____

https://duck.co/ia/view/vitamin_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/vitamin.json [hunk 1]
```diff
@@ -14,65 +14,65 @@
     "sections": {
         "Source": [{
                 "key": "Vitamin A",
-                "val": "Liver, fish, milk, cheese "
+				"val": "Cod, eggs, milk, meat, cheese, liver, sweet potatoes"
```
Comment (GuiltyDolphin) [inline share/goodie/cheat_sheets/json/vitamin.json]: The indentation seems to have got a bit messed up here.

Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/vitamin.json]: @GuiltyDolphin Thanks mate :+1:

File: share/goodie/cheat_sheets/json/vitamin.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
     "id": "vitamin_cheat_sheet",
     "name": "Vitamins",
-    "description": "Displays the source and benefits for each vitamin",
+    "description": "Displays the diatary source and benefits for each vitamin",
```
Comment (mintsoft) [inline share/goodie/cheat_sheets/json/vitamin.json]: dietary?

Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/vitamin.json]: :frowning:

**Comment by MrChrisW** [comment]
Fixed merge conflicts from #2741 :+1: @GuiltyDolphin This is good to go :rocket:

**Comment by GuiltyDolphin** [comment]
@MrChrisW Heheh, sorry about that :wink:

Anyway, checked this out again, looks good - thanks! :dolphin:

**Pull request merged**